### PR TITLE
BZ-1743299: Clarifying how to find what service accounts an SCC is ap…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…plied to

https://bugzilla.redhat.com/show_bug.cgi?id=1743299

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1743299/admin_guide/manage_scc.html#examining-a-security-context-constraints-object

### CODE REVIEW

File: admin_guide/manage_scc.adoc [hunk 1]
```diff
@@ -62,16 +62,17 @@ restricted         false     []        MustRunAs   MustRunAsRange     MustRunAs
 [[examining-a-security-context-constraints-object]]
 == Examining a Security Context Constraints Object
 
-To examine a particular SCC, use `oc get`, `oc describe`, or `oc
-edit`. For example, to examine the *restricted* SCC:
+You can view information about a particular SCC, including which users, service accounts, and groups that the SCC is applied to.
```
Comment (bobfuru) [inline admin_guide/manage_scc.adoc]: s/groups that the/groups the/

File: admin_guide/manage_scc.adoc [hunk 2]
```diff
@@ -98,6 +99,8 @@ Settings:
   Supplemental Groups Strategy: RunAsAny
     Ranges:				<none>
 ----
+<1> Lists which users and service accounts that the SCC is applied to.
```
Comment (bobfuru) [inline admin_guide/manage_scc.adoc]: s/accounts that the/accounts the

File: admin_guide/manage_scc.adoc [hunk 3]
```diff
@@ -98,6 +99,8 @@ Settings:
...
+<2> Lists which groups that the SCC is applied to.
```
Comment (bobfuru) [inline admin_guide/manage_scc.adoc]: s/groups that the/groups the/

**Review by bobfuru — COMMENTED** [review]
A few comments similar to the other PR

**Comment by bergerhoffer** [comment]
@barleyer Can you please review this? This to update the 3.9-3.11 docs.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1743299

**Pull request merged**